index.html

```diff
@@ -1305,6 +1307,94 @@ <h3>Verifiable Credential</h3>
 </pre>
     </section>
 
+    <section>
+      <h3>Verifiable Accreditation</h3>
+
+      <p>
+A <dfn>verifiable accreditation</dfn> is a recognition by an [=issuer=] that a
+specific entity is qualified to perform a particular action in an ecosystem. One
+example would be a national education board that recognizes colleges that are
+qualified to issue certain types of education degrees.
+      </p>
+
+      <p class="note"
+         title="Trust decisions are always local and can be overridden">
+Ecosystem participants are advised that all trust decisions are local and can be
+overridden. Different [=verifiers=] might use different accreditors and
+have risk profiles that might not require any accreditor at all. Further,
+some [=verifiers=] might have additional requirements that reject [=verifiable
+credentials=] even if they were issued by a trusted accreditor. Certain
+accrediting bodies can even be ignored, for legitimate reasons.
+      </p>
+
+      <section>
+        <h4>Issuing Accreditation</h4>
+
+        <p>
+A [=verifiable accreditation=] can be provided to an [=issuer=] to recognize it
+to issue certain types of [=verifiable credentials=]. A [=verifier=] would
+typically have a handful of accreditors that it trusts to vet [=issuers=]
+and would configure their verification software with that list of accreditors.
+Each accreditor would then issue a [=verifiable credential=] to
+each [=issuer=]. The [=issuer=] can then provide the [=verifiable
+accreditation=] to the [=holders=], who would then present the credential to the
+[=verifier=]. An example of such a [=verifiable credential=] issued by a trusted
+accreditor to an [=issuer=] is shown below. This system is similar in many
+ways to "traditional" X.509 Certificate Authorities, the certificates they
+issue, and the hierarchy of authority, authorization, and delegation.
+        </p>
+
+        <pre class="example nohighlight" title="An accreditation to issue education degrees">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "AccreditationCredential"
+  ],
+  "issuer": "did:web:accreditor.example",
+  "validFrom": "2025-01-01T00:00:00Z",
+  "validUntil": "2030-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:web:issuer.example",
+    "type": "Issuer",
```

Comment:
remove this line because it is covered by the TSP service type identifier

Reply:
I think the agreement was that the data model should be the same for both centralised and distributed lists.
So if you want to propose changes to the data model to address your example, please raise an issue for this. My changes use the same data for both examples, so this is the correct way to proceed.